tests/shared_src/test_main.xc

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016-2021 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#ifdef XUD_SIM_RTL
+int testmain()
+#else
+int main()
+#endif
+{
+    chan c_ep_out[EP_COUNT_OUT], c_ep_in[EP_COUNT_IN];
+
+    par
+    {
+        {
+#if defined(XUD_TEST_SPEED_FS)
+            unsigned speed = XUD_SPEED_FS;
```

Comment:
If speed is constant, define as `const`.

Reply:
Fixed